sub-logs/satindar.json

```diff
@@ -0,0 +1,17 @@
+{
+  "repo": "https://github.com/Satindar31/satindar.github.io",
+
+  "owner": {
+    "username": "Satindar31",
+    "email": "x7pz8871v@mozmail.com"
+  },
+
+  "target": {
+    "CNAME": {
+      "name": "satindar", 
+      "value": "satindar31.github.io"
```

Comment:
Just redirects to the domain which is requesting.

Reply:
Fixed but do I not need a CNAME file on my website(GitHub pages)?